docs/dev/build-instructions/build_linux.md

```diff
@@ -119,7 +116,6 @@ poco-devel \
 gcc-toolset-9-gcc-c++ 
 ```
 
-You will need `cmake` **v3.15 or later**. You can install a recent one from the [linked page](https://cmake.org/download/).
```

Comment:
This is still up-to-date, isn't it? 👀

Reply:
According to pkgs.org Centos 9 Stream ships with cmake >= 3.20

Reply:
Ah, I see :+1: